apps/edxapp/templates/services/nginx/route_cms.yml.j2

```
@@ -12,7 +12,9 @@ metadata:
     route_prefix: "{{ prefix }}"
     route_target_service: "cms"
   annotations:
+{% if prefix == 'current' or edxapp_nginx_disable_acme_on_next_and_previous is false %}
```

Implementation would look like:

```suggestion
{% if 'edxapp-nginx-cms-' + prefix not in acme_disabled_routes %}
```